frontend/src/App.js

```diff
@@ -42,7 +42,7 @@ export default function App() {
           <Container maxWidth="lg" sx={{ mt: 4, mb: 4 }}>
             <Routes>
               <Route path="/" element={<MetaInformationContainer />} />
-              <Route path="/image-viewer" element={<ImageViewerContainer />} />
+              <Route path="/image-viewer" element={<div style = {{display: "flex",flexDirection:"row"}}><ImageViewerContainer /><ImageViewerContainer /></div>} />
```

Comment:
Hier wäre ein Kommentar  toll. Vllt: 
// Renders the image viewer using flexbox layout with a row direction, allowing the ImageViewerContainer components to be displayed side by side horizontally